drwmutex.go

```diff
@@ -41,6 +41,7 @@ func log(msg ...interface{}) {
 
 // DRWMutexAcquireTimeout - tolerance limit to wait for lock acquisition before.
 const DRWMutexAcquireTimeout = 1 * time.Second // 1 second.
+const DRWMutexInfinite = time.Duration(1<<63-1)
```

Comment:
@balamurugana Well, technically it is not a timeout as it is (for all practical purposes) "infinitely" far away in the future... So I think adding Timeout would convey the wrong meaning here.